symphony-bdk-http/symphony-bdk-http-jersey2/src/main/java/com/symphony/bdk/http/jersey2/ApiClientJersey2.java

```diff
@@ -182,12 +162,29 @@ public ApiClientJersey2(final Client httpClient, String basePath, Map<String, St
             buildResponseHeaders(response),
             respBody);
       }
-    } finally {
-      try {
-        response.close();
```

Comment:
Why should we never close? I understand that in some cases, it does nothing but I'd like to know.

Reply:
@symphony-elias I used try -with-resources statement, so it is closed automatically (and btw a null check is performed before...so no risk of null pointer as it was before, even if silently intercepted)